_data/sites.json

```
@@ -6758,6 +6755,16 @@
         ]
     },
 
+    {
+        "name": "NordVPN",
+        "url": "https://support.nordvpn.com/FAQ/1521982312/How-can-I-delete-my-account.htm",
+        "difficulty": "hard",
+        "notes":"Fill the form on the site to request deletion.",
```

```suggestion
        "notes": "Fill the form on the site to request deletion.",
```
Small formatting nitpick